docs/index.rst

```diff
@@ -0,0 +1,69 @@
+.. Surfactant documentation master file, created by
+   sphinx-quickstart on Wed Oct 30 01:12:11 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. Surfactant documentation master file, created by
+   sphinx-quickstart on Mon Oct 28 15:43:14 2024.
+
+.. _Welcome to Surfactant’s documentation:
+
+=============================
+Welcome to Surfactant's Documentation!
+=============================
+
+.. image:: https://github.com/LLNL/Surfactant/actions/workflows/pytest.yml/badge.svg
+   :target: https://github.com/LLNL/Surfactant/actions/workflows/pytest.yml
+
+.. image:: https://img.shields.io/pypi/v/surfactant
+   :target: https://pypi.org/project/Surfactant/
+
+.. image:: https://img.shields.io/pypi/pyversions/surfactant.svg
+   :target: https://pypi.org/project/Surfactant/
+
+.. image:: https://img.shields.io/badge/License-MIT-blue.svg
+   :target: https://github.com/LLNL/Surfactant/blob/main/LICENSE
+
+.. image:: https://readthedocs.org/projects/surfactant/badge/?version=latest
+   :target: https://surfactant.readthedocs.io/en/latest/?badge=latest
+
+.. image:: https://results.pre-commit.ci/badge/github/LLNL/Surfactant/main.svg
+   :target: https://results.pre-commit.ci/latest/github/LLNL/Surfactant/main
+
+**Surfactant** is a modular framework for gathering file information and analyzing dependencies to generate an SBOM (Software Bill of Materials).
+
+Useful Links:
+-------------
+- `PyPI <https://pypi.org/project/Surfactant/>`_
+- `Source Code on GitHub <https://github.com/LLNL/Surfactant/>`_
+- `Discussions <https://github.com/LLNL/Surfactant/discussions/>`_
+
+.. toctree::
+   :maxdepth: 4
+   :caption: Contents
+
+   fileinfo
```

Comment:
The api reference links can be added to the list in this file instead: https://github.com/LLNL/Surfactant/blob/main/docs/index.md?plain=1#L34